### PR TITLE
Feature/apps 123 update course name 3

### PR DESCRIPTION
**Pull request by chen-c-walkme**

### CODE REVIEW

File: src/walkme/server/courseName/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import * as endpoint from '../endpoint';
+
+export async function updateCourseName(id: number, name: string): Promise<void> {
+  return endpoint.post('course/addOrUpdate', { course_id: id, course_name: name });
+}
```
Comment (nunibaranes) [inline src/walkme/server/courseName/index.ts:5]: Can you add please a newline at the end of the file?

File: src/walkme/server/endpoint/endpoint.ts [hunk 2]
```diff
@@ -2,8 +2,10 @@ import walkme from '@walkme/editor-sdk';
 
 const serviceDomain = 'https://api.walkme.com/teachmeservice';
 
-export const get = async (path: string): Promise<any> => {
-  const res = await fetch(`${serviceDomain}/${path}`, {
+export const fetch = async (path: string, method: string = 'GET', body?: any): Promise<any> => {
```
Comment (nunibaranes) [inline src/walkme/server/endpoint/endpoint.ts:5]: eslint error: Type string trivially inferred from a string literal, remove type annotation.
please remove the type annotation of the `method`.

File: src/walkme/server/endpoint/index.ts [hunk 3]
```diff
@@ -8,10 +8,16 @@ import { getLastUpdateTime } from './lastUpdate';
...
+  GET: 'GET',
+  POST: 'POST',
+};
+
 export const get = async (path: string): Promise<any> => {
   const lastUpdate = getLastUpdateTime();
-  const responseBody = await endpoint.get(
+  const responseBody = await endpoint.fetch(
     `${path}&last_update=${lastUpdate}&systemId=${_systemId}`,
+    METHODS.GET
```
Comment (nunibaranes) [inline src/walkme/server/endpoint/index.ts:20]: eslint error: Insert `,`

File: src/walkme/server/endpoint/index.ts [hunk 4]
```diff
@@ -22,6 +28,11 @@ export const get = async (path: string): Promise<any> => {
   return responseBody.data;
 };
 
+export const post = async (path: string, body: any) => {
+  return endpoint.fetch(path, METHODS.POST, JSON.stringify(body));
```
Comment (nunibaranes) [inline src/walkme/server/endpoint/index.ts:32]: esLint error: Unexpected block statement surrounding arrow body; move the returned value immediately after the `=>`

File: src/walkme/server/endpoint/index.ts [hunk 5]
```diff
@@ -22,6 +28,11 @@ export const get = async (path: string): Promise<any> => {
...
+};
+
+
```
Comment (nunibaranes) [inline src/walkme/server/endpoint/index.ts:35]: esLint error:  More than 1 blank line not allowed  
please remove this empty line

File: src/walkme/server/endpoint/lastUpdate.ts [hunk 6]
```diff
@@ -12,7 +12,7 @@ export function getLastUpdateTime(): number {
 async function fetchLastUpdateTime() {
   if (Date.now() - _lastUpdate.getTime() < TWO_HOURS_MILLISECONDS) return;
 
-  const { last_update_time } = (await endpoint.get('management/lastUpdate')) as ILastUpdateResponse;
+  const { last_update_time } = (await endpoint.fetch('management/lastUpdate')) as ILastUpdateResponse;
```
Comment (nunibaranes) [inline src/walkme/server/endpoint/lastUpdate.ts:15]: eslint error: Replace `'management/lastUpdate'` with `⏎····'management/lastUpdate',⏎··`

`const { last_update_time } = (await endpoint.fetch(
    'management/lastUpdate',
  )) as ILastUpdateResponse;`

**Review by nunibaranes — CHANGES_REQUESTED** [review]
looks great.
Please fix the eslint error

File: src/walkme/server/endpoint/index.ts (hunk 3 above)
Comment (chen-c-walkme) [reply src/walkme/server/endpoint/index.ts:20]: I'll add it now, but I hate this rule. Can we change it? @dr15

**Review by zuberikfir — APPROVED** [review]

**Pull request merged**